en/docs/guides/identity-federation/configure-ws-trust.md

```diff
@@ -46,141 +41,107 @@ To secure the Security Token Service:
-    <td><code>               15              </code></td>
+    <td>This is the duration in minutes for which an SSO session can be idle. If WSO2 Identity Server receives no SSO authentication requests for the given duration, a session time-out occurs. The default value is <code>15</code> .</td>
+    <td><code>15</code></td>
     </tr>
     <tr class="odd">
     <td><strong>Remember Me Period</strong></td>
     <td><div class="content-wrapper">
     <p>This is the duration in weeks for which WSO2 Identity Server should remember an SSO session given that the <strong>Remember Me</strong> option is selected in the WSO2 Identity Server login screen.</p>
-    <p>The default value is <code>                 2                </code> weeks.</p>
+    <p>The default value is <code>2</code> weeks.</p>
     </div></td>
-    <td><code>               2              </code></td>
+    <td><code>2</code></td>
     </tr>
     </tbody>
     </table>
 
-3.  Under the **Inbound Authentication Configuration** section, click
-    **Security Token Service Configuration \> Apply Security Policy**
-    .  
-    ![security-token-service-config]({{base_path}}/assets/img/tutorials/security-token-service-config.png)
-4.  Select **Yes** in the **Enable Security?** drop down and  select a
-    pre-configured security scenario according to your requirements. For
-    this tutorial, use **UsernameToken** under the **Basic Scenarios**
-    section.  
-    ![enable-security]({{base_path}}/assets/img/tutorials/enable-security.png)
+3. Expand **Inbound Authentication Configuration** > **Security Token Service Configuration** and click **Apply Security Policy**.
+    
+    ![apply security policy]({{base_path}}/assets/img/guides/ws-trust-apply-policy.png)
+
+4. Select `Yes` from the **Enable Security?** list and select a pre-configured security scenario according to your requirements.
+We will be using `UsernameToken` under the **Basic Scenarios** section for this tutorial.
 
     !!! note
-    
-        You can find further details about security policy scenarios from
-        the **view scenario** option **.**
-    
-        ![security-policy-scenarios]({{base_path}}/assets/img/tutorials/security-policy-scenarios.png)
-    
+        You can find further details about security policy scenarios from the **view scenario** option.
 
-5.  Click **Next**. The user domain and user group selection appears.
-	
-	!!! info	
-		Next steps may vary as per the security scenario that you have
-		chosen under point (5) above. Below is for **UsernameToken**
-		scenario.
+        ![enable-security]({{base_path}}/assets/img/guides/ws-trust-view-scenario.png)
 
-6.  Provide the required details as follows:
-    1.  Select **ALL-USER-STORE-DOMAINS**.
-    2.  Select the role you created to grant permission to access
-        secured service. In this example, the admin role is used.
-        Next, click **Finish**.
+5. Click **Next**.
```

Comment:
Step 5 and 6 can be combined isn't it.

`Click Next and provide the required details as follows:`